openid-connect-4-credential-issuance-1_0.md

```diff
@@ -347,15 +436,71 @@ Below is a non-normative example of a `claims` parameter including the `credenti
 }
 ```
 
+### Pushed Authorization Request
+
+This step is OPTIONAL. In case of complex flows or when the size of the request is large due to the query language or encrypting, clients are RECOMMENDED to use pushed uthorization requests.
```

Comment:
We should always recommend to use PAR. It protects the claims parameter from modifications.